src/multiboot-headers.md

```diff
@@ -282,7 +282,21 @@ Again, we’ll use math to let the computer calculate the sum for us. We add up
 the magic number, the mode code, and the header length, and then subtract it
 from a big number. `dd` then puts that value into this spot in our file.
 
-TODO: https://github.com/intermezzOS/book/issues/28
+> By the way...
+>
+> 0xe85250d6 is 3,897,708,758 in decimal, and the smallest number you can store as a 32-bit signed integer is -2,147,483,648, so we get an integer underflow. This doesn't matter for the checksum, but since underflows are usually a bug the assembler gives us a warning.
```

Comment:
For the record, here's what I wrote on signed integers:

https://gist.github.com/dariusk/505305e584bd8a2006f5

I'm putting it here so that there's a record of it. We might need it later on in the book even if it isn't useful right here.